sonar-db/src/main/java/org/sonar/db/permission/PermissionTemplateDao.java

```diff
@@ -296,20 +296,28 @@ public void insertGroupPermission(DbSession session, Long templateId, @Nullable
     session.commit();
   }
 
+  /**
+   * @deprecated since 5.2 use the method with DbSession
+   */
+  @Deprecated
   public void deleteGroupPermission(Long templateId, @Nullable Long groupId, String permission) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1133)